### PR TITLE
Springboot Config

### PR DESCRIPTION
**Pull request by eetar1**

### CODE REVIEW

File: web/backend/v2/gradlew [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    else
+        PRG=`dirname "$PRG"`"/$link"
+    fi
+done
+SAVED="`pwd`"
+cd "`dirname \"$PRG\"`/" >/dev/null
+APP_HOME="`pwd -P`"
+cd "$SAVED" >/dev/null
+
+APP_NAME="Gradle"
```
Comment (Ross-Alexandra) [inline web/backend/v2/gradlew]: Should this be "TourneyGen" not sure how gradle works exactly.

File: web/backend/v2/src/test/java/com/tourneygen/web/WebApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.tourneygen.web;
```
Comment (Ross-Alexandra) [inline web/backend/v2/src/test/java/com/tourneygen/web/WebApplicationTests.java]: The filenames here are getting really long. Can we cut this down to web/backend/v2/com/.. ? or at least reduce the number of seemingly redundant directories here?

**Comment by Ross-Alexandra** [comment]
Thinking out loud @eetar1 could you actually move the v1, v2 business up inside src (so we have src/v1 and src/v2) so that we can build out a v2 of the frontend as well in the same structure (making for an easier switch over to v2 when it's ready for release)?

File: web/backend/v2/gradlew (hunk 1 above)
Comment (eetar1) [reply web/backend/v2/gradlew]: Neither am i, but i set the app name in config so i leave this alone

Comment (Ross-Alexandra) [reply web/backend/v2/gradlew]: Fair enough!

File: web/backend/v2/src/test/java/com/tourneygen/web/WebApplicationTests.java (hunk 2 above)
Comment (eetar1) [reply web/backend/v2/src/test/java/com/tourneygen/web/WebApplicationTests.java]: this was all generated by the framework. Messing with it breaks gradle lets just leave it until it causes a problem. Dirs easy to fix

**Comment by Ross-Alexandra** [comment]
@eetar1 This one's my bad, I wasn't looking at the structure when I left the comment, I meant to have v1 and v2 in web so we have web/v1/frontend, web/v1/backend, web/v2/frontend, and web/v2/backend

Send a text or something more immediate when you're done so I can look at it quicker.

**Pull request merged**